react/components/GalleryItem.js

```diff
@@ -52,17 +52,33 @@ const GalleryItem = ({ item, displayMode, summary }) => {
 
   const { push } = usePixel()
 
-  const sendProductClickEvent = product => {
+  const pushPixelProductClickEvent = product => {
+    product.selectedSku = product.sku.itemId
```

Comment:
I don't  think it is safe to change it like this:

it would be safe to do:
```
const selectedProduct = {
  ...product,
  selectedSku: product.sku.itemId,
}
```

Reply:
Thanks! I really appreciated this tip. But now I think the best place to do it is where the events will be received. So I deleted this line.